flakeModules/cluster.nix

```diff
@@ -427,6 +429,12 @@ flake @ {
         default = [];
       };
 
+      blockfrost-platform-service = mkOption {
```

Comment:
Style/Fmt: can we keep these code blocks alphabetical?  2 blocks in this file, 1 block in the pkgs.nix file.